### PR TITLE
feat(languages): rescript

### PR DESCRIPTION
**Pull request by jaredramirez**
Hey there,

This PR adds support for the [rescript](https://rescript-lang.org/) language!

### CODE REVIEW

**Comment by EpocSquadron** [comment]
Are you proficient enough to add textobject queries? Should be a fairly easy add.

You're also missing a line ending in the languages.toml.

**Review by EpocSquadron — COMMENTED** [review]
Change `.inner` and `.outer` to `.inside` and `.around`, respectively, as those are the capture names helix expects.

**Comment by EpocSquadron** [comment]
You'll also need to rerun xtask to pick up the textobject support in the language support table.

File: runtime/queries/rescript/textobjects.scm [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+; Functions
+;----------
+
+(function body: (_) @function.inner) @function.outer
+
+; Parameters
+;-----------
+
+(function parameter: (_) @parameter.inner @parameter.outer)
```
Comment (EpocSquadron) [inline runtime/queries/rescript/textobjects.scm:14]: You can remove parameter.outer/around here as it's not used and wild be differently queued if it were

**Comment by EpocSquadron** [comment]
Almost there! Sorry to be nagging, but it's "around" instead of "outside". I'm working on a screencast walkthrough of how these queries work, I regret that it wasn't ready to give to you up front and make this easier on you.

**Comment by jaredramirez** [comment]
No worries, I totally missed that it was supposed to be `around`! Let me know if there's anything else needed!

**Comment by EpocSquadron** [comment]
LGTM

**Pull request merged**